aidb_utilities/db_setup/create_tables.py

```diff
@@ -1,42 +1,45 @@
-import sqlalchemy.ext.asyncio
-from sqlalchemy.schema import ForeignKeyConstraint
+from typing import Dict
 
 import sqlalchemy
 import sqlalchemy.ext.asyncio
 from sqlalchemy import MetaData
+from sqlalchemy.schema import ForeignKeyConstraint
 
 from aidb.config.config_types import python_type_to_sqlalchemy_type
 from aidb.utils.db import create_sql_engine
 
 
-async def create_output_tables(db_url: str, db_name: str, tables_info):
-  db_uri = f'{db_url}/{db_name}'
+async def create_output_tables(db_config: Dict[str, str], output_tables: Dict):
```

Comment:
why did the type signature change?

Reply:
Since both db_url and db_name come from db_config, I merged them into a dictionary. For output_tables, I realized that the type should be List[dict]. The previous code only allowed a single blob table, using a dictionary for output table configuration and a list for service configuration. I changed them all to lists, as multiple items are allowed.